### PR TITLE
Download links in Display Settings [stage-1]

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Thanks

### CODE REVIEW

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -112,15 +112,19 @@
 
 <div class="form-group form-inline" ng-show="displayId">
   <label class="control-label" translate>displays-app.fields.id</label>
+  <a ng-show="displayId" class="btn btn-link pull-right" href="https://help.risevision.com/user/create-a-display#download-rise-player">
```
Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @ezequielc I think this needs to open in a new window. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good, one comment re opening the help doc in a new window. Thanks.

File: web/partials/displays/display-fields.html (hunk 1 above)
Comment (ezequielc) [inline web/partials/displays/display-fields.html]: @alex-deaconu Good point. I will do that. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Thanks for fixing :+1: .

**Pull request merged**